make target all fails on osx

Trying to make on `osx` fails.

```
 make all
GOARCH=amd64 GO111MODULE=on go build -mod=vendor ./...
GOARCH=386 GO111MODULE=on go build -mod=vendor ./...
cmd/go: unsupported GOOS/GOARCH pair darwin/386
make: *** [build-386] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -58,7 +58,9 @@ build-amd64:
 
 .PHONY: build-386
 build-386:
+ifneq ($(shell uname -s), Darwin)
 	GOARCH=386 $(GO_BUILD) ./...
+endif
 
 .PHONY: docs
 docs:

